common/network-common/src/main/java/org/apache/spark/network/util/TransportConf.java

```diff
@@ -257,4 +257,7 @@ public Properties cryptoConf() {
     return CryptoUtils.toCryptoConf("spark.network.crypto.config.", conf.getAll());
   }
 
+  public long maxChunksBeingTransferred() {
+    return conf.getLong("spark.network.shuffle.maxChunksBeingTransferred", Long.MAX_VALUE);
```

Comment:
It depends on memory config of NodeManager. In our cluster, we will set it to be 3G(memory overhead for Netty)/1k(size of ChannelOutboundBuffer$Entry)=3000000.
I think it's ok to leave it MAX_VALUE(by default it's disabled).